{"name": "Test login with invalid username", "status": "passed", "description": "This test verifies that an error message is displayed when logging in with an invalid username.", "attachments": [{"name": "Invalid Username - Error Message", "source": "8d162d69-9900-4a5b-b621-7d7cd46ce156-attachment.png", "type": "image/png"}], "start": 1740404459979, "stop": 1740404462780, "uuid": "638f336a-1306-44fa-a23b-ca9ab1acb732", "historyId": "8dfefa9403d2f0167e5ccac7cf061037", "testCaseId": "8dfefa9403d2f0167e5ccac7cf061037", "fullName": "test_form_authentication.TestFormAuthentication#test_invalid_username", "labels": [{"name": "story", "value": "Invalid Login"}, {"name": "feature", "value": "Form Authentication"}, {"name": "suite", "value": "test_form_authentication"}, {"name": "subSuite", "value": "TestFormAuthentication"}, {"name": "host", "value": "Jonathan"}, {"name": "thread", "value": "19388-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_form_authentication"}]}